doc/train/train-input.rst

```diff
@@ -2,5 +2,18 @@ Training Parameters
 ======================================
 .. note::
    One can load, modify, and export the input file by using our effective web-based tool `DP-GUI <https://deepmodeling.org/dpgui/input/deepmd-kit-2.0>`_. All training parameters below can be set in DP-GUI. By clicking "SAVE JSON", one can download the input file for furthur training.
+   
+   
+   **Available activation functions for descriptor:**
```

Comment:
Why insert here?

Reply:
Because there is a lack of guidance on available activation functions, neither in DP-GUI nor for editing 'input.json' by oneself. 
You can tell me if there is a more suitable place to add.

Reply:
I think it's mentioned, though.

![image](https://user-images.githubusercontent.com/9496702/141412783-4e9ff7c4-d584-4d1c-997c-3d0b04e28a94.png)

Reply:
Fine~ It's better to left document work to you because it‘s hard for me to think about it comprehensively.